layouts/community/75_ansi/spidey3/keymap.c

```diff
@@ -0,0 +1,36 @@
+#include QMK_KEYBOARD_H
+#include "unicode.h"
+#include "spidey3.h"
+
+#define SETTINGS (A(S(KC_S)))
```

Comment:
Mostly just a nit pick: 
```suggestion
#define SETTINGS A(S(KC_S))
```